tools/c7n_left/c7n_left/core.py

```diff
@@ -59,6 +59,10 @@ def description(self):
     def display_category(self):
         return " ".join(self.categories)
 
+    @property
+    def url(self):
+        return self.policy.data.get("metadata", {}).get("url")
```

Comment:
nit: the other metadata properties seem to fall back to `""` to ensure strings. necessary here?

Reply:
we do a filter empty on where url gets used as its not required, and actually severity should probably be unknown as a default instead of empty string.